lib/Conversion/TorchOnnxToTorch/DefaultDomainGtoP.cpp

```diff
@@ -1894,6 +1894,36 @@ void mlir::torch::onnx_c::populateDefaultDomainGtoP(
         rewriter.replaceOpWithNewOp<Torch::AtenOneHotOp>(binder.op, resultType,
                                                          argmax, oneInt);
 
+        return success();
+      });
+  patterns.onOp(
+      "LpNormalization", 22,
+      [](OpBinder binder, ConversionPatternRewriter &rewriter) {
+        Torch::ValueTensorType resultType;
+        int64_t axis, p;
+        Value input;
+        if (binder.tensorOperand(input) ||
+            binder.s64IntegerAttr(axis, "axis", -1) ||
+            binder.s64IntegerAttr(p, "p", 2) ||
+            binder.tensorResultType(resultType))
+          return failure();
+
+        auto loc = binder.getLoc();
+        Value cstAxis = rewriter.create<Torch::ConstantIntOp>(
+            loc, rewriter.getI64IntegerAttr(axis));
+        Value cstP = rewriter.create<Torch::ConstantIntOp>(
```

Comment:
Do we know if AtenNormScalarOptDimOp handles negative dims?

Reply:
Yes, it's handled here: https://github.com/llvm/torch-mlir/blob/e4be197efd85916cd378ef8e7f21ca3de13b5903/lib/Conversion/TorchToLinalg/Reduction.cpp#L480-L485